ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12-periodics.yaml

```diff
@@ -0,0 +1,80 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 0 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: community-4.12
+    org: openshift
+    repo: windows-machine-config-operator
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: okd
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-windows-machine-config-operator-community-4.12-okd-aws-e2e-operator
+  reporter_config:
+    slack:
+      channel: '#forum-winc'
+      job_states_to_report:
+      - failure
+      report_template: Job {{.Spec.Job}} failed. Do not use this bundle for the next
+        community WMCO release.
```

Comment:
Is there a way to indicate the commit hash?

Reply:
Maybe it could be added as a variable in the job, but I'm not sure how I'd pull it in here. Need to think about this more

Reply:
Did you take a look at [upstream documentation](https://godoc.org/k8s.io/test-infra/prow/apis/prowjobs/v1#ProwJob) mentioned in [Set Up Slack Alerts for Periodic Job Results](https://docs.ci.openshift.org/docs/how-tos/notification/)?

Reply:
In addition to the Job id, it would be helpful to include a link to the failed job in the slack message.  You could use `<{{.Status.URL}}|View logs>` in the message.  Maybe also mention `@winc-watcher` so they can investigate the failure.  You can look through the other jobs in the release repo to find more inspiration for crafting the slack message.

Here is a suggestion.  You can incorporate any of this into your message.
```
report_template: ':warning: <!subteam^S025TMX0LF3> Job *{{.Spec.Job}}* failed. <{{.Status.URL}}|View logs> :warning:'
```

I looked up the formatting for [mentioning groups](https://api.slack.com/reference/surfaces/formatting#mentioning-groups) in Slack and came up with `<!subteam^S025TMX0LF3>` for `@winc-watcher`.